components/prism-iecst.js

```diff
@@ -0,0 +1,40 @@
+Prism.languages.iecst = {
+	'comment': [
+		{
+			pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+			lookbehind: true
+		},
+		{
+			pattern: /(^|[^\\])\(\*[\s\S]*?(?:\*\)|$)/,
+			lookbehind: true
+		},
+		{
+			pattern: /(^|[^\\])\{[\s\S]*?(?:\}|$)/,
+			lookbehind: true
+		},
```

Comment:
It's more efficient to combine these patterns.

```suggestion
		{
			pattern: /(^|[^\\])(?:\/\*[\s\S]*?(?:\*\/|$)|\(\*[\s\S]*?(?:\*\)|$)|\{[\s\S]*?(?:\}|$))/,
			lookbehind: true
		},
```